src/main/daml/Daml/Finance/Instrument/Bond/Util.daml

```diff
@@ -51,11 +54,44 @@ includes : Schedule -> Schedule -> [Bool]
 includes scheduleA scheduleB =
   map (\a -> isSome $ find (\b -> b.adjustedEndDate == a.adjustedEndDate) scheduleB) scheduleA
 
+-- | Calculate the fixing date by using a day shift (if applicable) and then adjusting the date.
```

Comment:
These helper functions have been extracted from `createCallableBondClaims`. Shall we prevent them from being exported or `-- | HIDE` them?